conventions.md

```diff
@@ -1,3 +1,58 @@
 # Cryptic Conventions
-You can find the conventions in our official Wiki:
-- [Konventionen (Deutsch)](https://wiki.cryptic-game.net/books/einf%C3%BChrungen/page/konventionen)
+
+ Please use English for all Cryptic-related GitHub posts (in issues, pull requests, commits, etc) and follow the conventions below to ensure a clear and uniform style.
+
+## Branch Naming
+
+The master (or main) branch contains the actual version. New commits to are only added via pull requests which requires two approvals. Learn more about [GitHub branches](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches).
+
+**Feature branches** contain one or more commits to address one related feature or issue. Prefix your feature branch with `issue/issue-id` before you add the actual branch name. Example: `issue/\<id\>-\<add-my-feature\>`. To update any project dependencies, name your branch like `dependencies/\<dependencies-name\>`.
+
+## Commit Messages
+
+Write short (< 50 characters) **commit messages** in English and use the scheme `<message> (#<issue-id>)`. Do not use just `<em>bug fix</em>` but useful text.
+
+The commit message that fixes any merge conflicts in your feature branch (after you have merged master into your branch) should be `Solved merge conflicts for #<pull-request-id>`.
+
+<!-- special hotfix commit msg? -->
+
+### Merging Branches
+
+To merge a feature branch into master, use the option **merge** (not squash or rebase) to create a merge commit.
+
+## Issues and Pull Requests
+
+- **Title**: Shortly summarize the content but avoid special characters.
+- **Content/Description**: Add more information to the issue or pull request.
+
+When submitting an issue or pull request, fill out all other information requested in these predefined templates:
+
+- [Bug Report](https://github.com/cryptic-game/cryptic/blob/master/.github/ISSUE_TEMPLATE/BUG_REPORT.md)
+- [Feature Request](https://github.com/cryptic-game/cryptic/blob/master/.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md)
+- [Pull Request Template](https://github.com/cryptic-game/cryptic/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
+
+## Release Versioning / Hotfixes
+
+For releases, we follow [Semantic Versioning 2.0.0](https://semver.org/).
+
+Heads (or Head Assistants) can create/approve hotfixes for severe/security-related issues to address quick fixes. Such commits should use the naming scheme: `Hotfix: <message>` (z.B. `Hotfix: Changed foo bar`). Create a new branch from a release-tag and name the new release as `<old-release-name>.x` - each hotfix for a release increases x by 1, e.g. `v0.3.1-pre-alpha.2`.
```

Comment:
Heads (or Head Assistants) is a pretty internal information. Not sure if this need to be exposed to public.